tensorflow/docs_src/extend/index.md

```diff
@@ -20,8 +20,9 @@ TensorFlow:
 
 Python is currently the only language supported by TensorFlow's API stability
 promises.  However, TensorFlow also provides functionality in C++, Java, and Go,
-plus community support for [Haskell](https://github.com/tensorflow/haskell)
-and [Rust](https://github.com/tensorflow/rust).  If you'd like to create or
+plus community support for [Haskell](https://github.com/tensorflow/haskell), 
+[Scala](https://github.com/eaplatanios/tensorflow_scala), and 
```

Comment:
Let's revert the change in this file. The languages listed here are maintained by members of the TensorFlow github organization, just don't get the same level of support. I'd like for documentation to be clear between bindings maintained by (and for which support can be expected from) the TensorFlow organization and those owned, maintained and supported by others.